shelley/chain-and-ledger/executable-spec/bench/Main.hs

```diff
@@ -131,7 +162,7 @@ main :: IO ()
 main =
   defaultMain $
     [ bgroup "vary input size" $
-        [ varyInput "deregister key" (1, 5000) [(1, 50), (1, 500), (1, 5000)] ledgerStateWithNregisteredKeys ledgerDeRegisterStakeKeys,
+        [ -- varyInput "deregister key" (1, 5000) [(1, 50), (1, 500), (1, 5000)] ledgerStateWithNregisteredKeys ledgerDeRegisterStakeKeys,
```

Comment:
The current code is too slow to run at the size all the other tests run [50,500,5000]. This needs a BiMap so that computing range restriction is no O(n .* log n). This should be in the next  PR. So I would like to leave the comment code in til then. I will add a comment to explain why it is left in.